Find a functionally equivalent code:android.os.Bundle.getClassLoader:COMMENT
Method Modifier: public      
Comment:/**
 * Return the ClassLoader currently associated with this Bundle.
 */

@@ -1,3 +1,3 @@
 {
-    return mClassLoader;
+    return super.getClassLoader();
 }

